src/pkg/services/m365/api/graph/http_wrapper.go

```diff
@@ -193,12 +193,28 @@ func internalMiddleware(
 		counter: counter,
 	}
 
+	// We use default kiota retry handler for 503 and 504 errors
+	kiotaRetryHandler := khttp.NewRetryHandlerWithOptions(
+		khttp.RetryHandlerOptions{
+			ShouldRetry: func(
```

Comment:
huh, kinda a bummer they make you give a trivial function, but I guess that's how it is

Reply:
haha! yeah